scripts/upgrade

```diff
@@ -70,50 +87,79 @@ if ynh_version_gt "1.07~ynh2" "${previous_version}" ; then
    ynh_install_app_dependencies "mongodb mongodb-server"
    yunohost service remove mongod
    yunohost service add mongodb --log "/var/log/mongodb/mongodb.log"
+   ynh_use_nodejs
```

Comment:
Are you sure it's needed here since that's done later right before the npm install ?

Reply:
In fact, it's needed two line below for $nodejs_path
I reorder the lines to be more understandable